pkg/server/sandbox_status_test.go

```diff
@@ -52,9 +52,10 @@ func TestPodSandboxStatus(t *testing.T) {
 		Annotations: map[string]string{"c": "d"},
 	}
 	metadata := sandboxstore.Metadata{
-		ID:     id,
-		Name:   "test-name",
-		Config: config,
+		ID:             id,
+		Name:           "test-name",
+		Config:         config,
+		RuntimeHandler: "gvisor",
```

Comment:
Change all this to `test-runtime-handler`.

We may not want to include specific runtime name in our unit test. :)

Reply:
Done!